src/04-modality-specific-files/03-electroencephalography.md

```diff
@@ -279,7 +279,7 @@ Template:
 sub-<label>/
     [ses-<label>]/
       eeg/
-        [sub-<label>[_ses-<label>][_acq-<label>][_run-<index>]_electrodes.tsv]
+        sub-<label>[_ses-<label>][_acq-<label>][_space-<label>]_electrodes.tsv]
```

Comment:
```suggestion
        sub-<label>[_ses-<label>][_acq-<label>][_space-<label>]_electrodes.tsv
```

Reply:
Why is the extra bracket needed?

Reply:
It's not needed, it's there by mistake. The square brackets are meant to indicate optional versus required entities :)

Reply:
Ah I see, I think I confused myself ... :p